From 537067248021228861ec88bb1feb8922e9818701 Mon Sep 17 00:00:00 2001
From: Roland McGrath <roland@hack.frob.com>
Date: Fri, 2 Sep 2016 16:56:35 -0700
Subject: [PATCH] NaCl: Fix compile error in clock function.

	* sysdeps/nacl/clock.c (clock): nacl_abi_clock_t -> nacl_irt_clock_t

(cherry picked from commit 307c2c2dfff76330a29a3ab69a0177b118142145)
---
 ChangeLog            | 4 ++++
 sysdeps/nacl/clock.c | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/ChangeLog b/ChangeLog
index 3af5852df4ee..01272446091e 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,7 @@
+2016-09-02  Roland McGrath  <roland@hack.frob.com>
+
+	* sysdeps/nacl/clock.c (clock): nacl_abi_clock_t -> nacl_irt_clock_t
+
 2016-08-17  Florian Weimer  <fweimer@redhat.com>
 
 	Reduce time to expected nptl/tst-once5 failure.
diff --git a/sysdeps/nacl/clock.c b/sysdeps/nacl/clock.c
index 664ad650c3f6..b6fbcfd2dd0a 100644
--- a/sysdeps/nacl/clock.c
+++ b/sysdeps/nacl/clock.c
@@ -24,6 +24,6 @@
 clock_t
 clock (void)
 {
-  nacl_abi_clock_t result;
+  nacl_irt_clock_t result;
   return NACL_CALL (__nacl_irt_basic.clock (&result), result);
 }
-- 
2.10.2

